internal/pkg/manifest/validate.go

```diff
@@ -843,6 +845,15 @@ func (ip IPNet) validate() error {
 
 // validate returns nil if NetworkLoadBalancerConfiguration is configured correctly.
 func (c NetworkLoadBalancerConfiguration) validate() error {
+	for idx, listener := range c.NLBListeners() {
```

Comment:
Do we want to validate here that `AdditionalListeners` are there only if `MiainListener` is there?